server/config.js

```diff
@@ -211,5 +211,5 @@ module.exports = {
     ),
   },
 
-  riskManagementVersion: get('LICENCE_RISK_MANAGEMENT_VERSION', 2),
+  riskManagementVersion: get('LICENCE_RISK_MANAGEMENT_VERSION', '2'),
```

Comment:
Turns out this fallback risk value is being used in integration tests on circleci and the previous integer 2 was failing the strict equals conditional check in the risk management route GET (2 does not === '2' therefore the v2 page risk management page was not being displayed). Fixed the failing integration tests with this change.